### PR TITLE
Add the initial build pipeline skeleton

### PR DESCRIPTION
**Pull request by yufeih**
cc #2630

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -1,43 +0,0 @@
-# DocFX v3 Roadmap
```
Comment (vicancy) [inline docs/roadmap.md]: >Roadmap [](start = 11, length = 7)

place a link to the project dashboard rather then delete?

File: test/docfx.Test/docfx.Test.csproj [hunk 2]
```diff
@@ -4,20 +4,11 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <IsPackable>false</IsPackable>
     <LangVersion>Latest</LangVersion>
-    <CodeAnalysisRuleSet>../../stylecop.ruleset</CodeAnalysisRuleSet>
```
Comment (vicancy) [inline test/docfx.Test/docfx.Test.csproj]: >ruleset [](start = 40, length = 7)

does it mean no styles for Tests? Why?

Comment (yufeih) [reply test/docfx.Test/docfx.Test.csproj]: tests tends to generate a lot of stylecop/fxcop warnings. see [https://ci.appveyor.com/project/DocFXService/docfx/build/1.0.114](https://ci.appveyor.com/project/DocFXService/docfx/build/1.0.114)

---
In reply to: [181612693](https://github.com/dotnet/docfx/pull/2634#discussion_r181612693) [](ancestors = 181612693)

File: docs/roadmap.md (hunk 1 above)
Comment (yufeih) [reply docs/roadmap.md]: README.md is updated to point to project dashboard.

---
In reply to: [181612382](https://github.com/dotnet/docfx/pull/2634#discussion_r181612382) [](ancestors = 181612382)

**Pull request merged**